org.eclipse.jdt.core.manipulation/common/org/eclipse/jdt/internal/ui/util/ResourcesUtility.java

```diff
@@ -0,0 +1,141 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
```

Comment:
Same here.  Add Red Hat line about moving logic from XX to here.  If too many files, put in line comments below.